dags/vm_resource.py

```diff
@@ -188,11 +188,9 @@ class DockerImage(enum.Enum):
       f"{datetime.datetime.today().strftime('%Y-%m-%d')}"
   )
   MAXTEXT_TPU_JAX_SS = (
-      "gcr.io/tpu-prod-env-multipod/jax-ss-maxtext-unpinned:06032024"
-  )
-  MAXDIFFUSION_TPU_JAX_SS = (
-      "gcr.io/tpu-prod-env-multipod/maxdiffusion_jax_ss:06112024"
+      "gcr.io/tpu-prod-env-multipod/maxtext-jax-ss:jax0.4.30-rev1-07172024"
   )
+  MAXDIFFUSION_TPU_JAX_SS = "gcr.io/tpu-prod-env-multipod/maxdiffusion-jax-ss/tpu:jax0.4.30-rev1-07172024"
```

Comment:
Since the tag is fixed, we'll be testing the same code each execution (and this change adds a lot of tests) - should we wait to merge until there's a nightly ss build?

Reply:
@jonb377 Done. I have introduced daily builds of M* jax stable stack workloads and updated the code to reflect the same. PTAL.